tests/Equinox.CosmosStore.Integration/AccessStrategies.fs

```diff
@@ -133,3 +140,33 @@ type UnoptimizedTipReadingCorrectness(testOutputHelper) =
 
         let! s2'' = service2.Read instanceId
         test <@ s1'' = s2'' @> }
+
+
+module Token =
+    #if STORE_DYNAMO
+    let getPos index = { index = index; etag = ""; calvedBytes = 0; baseBytes = 0; unfoldsBytes = 0; events = Array.empty }
+    #else
+    let getPos index = { index = index; etag = None }
+    #endif
+    let [<Fact>] ``Candidate is not stale if we have no current`` () =
+        let emptyToken = Unchecked.defaultof<StreamToken>
```

Comment:
I wonder if this can be triggered by having L134 have a variant that passes Equinox.LoadOption.AnyCachedValue or similar (but will focus on getting this released instead!)